com.google.Chrome.desktop

```diff
@@ -0,0 +1,34 @@
+[Desktop Entry]
+Name[hi]=Google Chrome
+Name[th]=Google Chrome
+Name[ro]=Google Chrome
+Comment[hi]=Google का लोकप्रिय वेब ब्राउज़र
+Comment[fr]=Le navigateur Web populaire de Google
+Name[pt_BR]=Google Chrome
+Comment[pt_BR]=Navegador popular do Google
+Name[es]=Google Chrome
+Comment[id]=Peramban web populer dari Google
+Name[ar]=Google Chrome
+Exec=/app/bin/eos-google-chrome-app
+Version=1.0
+Comment[vi]=Trình duyệt web phổ biến nhất của Google
+Type=Application
+Icon=com.google.Chrome
```

Comment:
Can you also add our custom icon to the EndlessOS flatpak icon theme extension?

Reply:
https://github.com/endlessm/eos-theme/pull/298